packages/components/src/components/video/video.ts

```diff
@@ -0,0 +1,137 @@
+ * @summary Used to wrap external video elements (e. g. from Moving Image or bare <video>-Tags) and provide some basic styling for Union Investment.
+ * @status stable
+ * @since 1.19.0
+ *
+ * @dependency sd-example
+ *
+ * @event sd-play - Event emitted when clicking the `play-icon`. Listen for this event and use it to play the wrapped video.
+ *
+ * @slot - The default slot used to pass a video player element.
+ * @slot play-icon - The video's play icon.
+ * @slot poster - Specifies an image to be shown before initial play of the wrapped video. Acts like the 'poster' attribute on the native <video> tag. Optionally use this slot to mask
+ * loading appearance of wrapped video player.
+ *
+ * @csspart base - The component's base wrapper.
+ * @csspart play-button - Button element wrapper around the play-icon slot.
+ */
+
+@customElement('sd-video')
+export default class SdVideo extends SolidElement {
+  /** Set to `true` to hide the play icon and the overlay. */
+  @property({ type: Boolean, reflect: true }) playing = false;
+
+  /** Set to `true` to show a dark overlay. Only used when `playing` is `false`. */
+  @property({ type: Boolean, reflect: true }) overlay = false;
+
+  private readonly hasSlotController = new HasSlotController(this, '[default]', 'play-icon', 'poster');
+
+  /** Getter for optional poster slot. */
+  private get poster(): Element | null {
+    const slot: HTMLSlotElement = this.shadowRoot!.querySelector('slot[name=poster]')!;
+
+    if (slot?.assignedElements().length > 0) {
+      return slot.assignedElements()[0];
+    }
+    return null;
+  }
+
+  /** Method to fade out poster after initial play. */
+  private fadeoutPoster(): void {
+    if (this.poster instanceof HTMLImageElement) {
+      this.poster.style.opacity = '0';
+    }
+  }
+
+  /** Method to hide poster after initial play & fadeout. */
+  private hidePoster() {
+    if (this.poster instanceof HTMLImageElement) {
+      this.poster.style.display = 'none';
+    }
+  }
+
+  /** Internal method to group play behaviors. If a KeyboardEvent is used, refocus on the native video element to give the user seamless keyboard control. */
+  private play(e: MouseEvent | KeyboardEvent) {
+    this.emit('sd-play');
+    this.playing = true;
+    this.fadeoutPoster();
+
+    if (e instanceof KeyboardEvent) {
+      setTimeout(() => {
+        this.querySelector('video')?.focus();
+      });
+    }
+  }
+
+  render() {
+    return html`
+      <div part="base" aria-label="Video Player">
+        ${this.hasSlotController.test('poster')
+          ? html`<slot name="poster" part="poster" role="presentation" @transitionend=${this.hidePoster}></slot>`
+          : null}
+        <div
+          part="overlay"
+          id="overlay"
+          role="presentation"
+          class=${cx(
+            this.overlay && !this.playing ? 'opacity-100' : 'opacity-0',
+            'bg-[rgba(0,0,0,0.65)] w-full h-full absolute top-0 left-0 pointer-events-none z-20 play-pause-transition'
```

Comment:
```suggestion
            'bg-black/65 w-full h-full absolute top-0 left-0 pointer-events-none z-20 play-pause-transition'
```

Couldn't you use something like bg-black/65 or something?

Reply:
discussed via phone --- no bg-black token exists :(